### PR TITLE
add admin-only teenager indicator to user settings

### PR DESCRIPTION
**Pull request by mrhappyma**
add an admin-only teenager indicator to user settings
closes #8428 

image

hopefully i did this right?

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -170,11 +170,18 @@
...
+                                 start_year: Date.today.year,
+                                 end_year: Date.today.year - 100,
+                                 order: [:month, :day, :year],
+                                 prompt: @user.birthday.nil? %>
+            <% if admin_signed_in? && !@user.birthday.nil? %>
+              <p class="admin-tools">
+                <%= @user.birthday > 19.years.ago ? "they're a teenager ✅" : "they're not a teenager" %>
+              </p>
+            <% end %>
+          </div>
```
Comment (sampoder) [inline app/views/users/edit.html.erb]:
```suggestion
          <div class="flex items-center space-x-1">
            <%= form.date_select :birthday,
                                 start_year: Date.today.year,
                                 end_year: Date.today.year - 100,
                                 order: [:month, :day, :year],
                                 prompt: @user.birthday.nil? %>
            <% unless @user.birthday.nil? %>
              <% admin_tool do %>
                <%= @user.birthday > 19.years.ago ? "✅ Teenager" : "Not a teenager" %>
              <% end %>
            <% end %>
          </div>
```

We have a helper called `admin_tool` to generate admin tools that are only shown to admins

Comment (sampoder) [reply app/views/users/edit.html.erb]: Thanks for making this PR btw!

**Review by sampoder — APPROVED** [review]
LGTM!

**Comment by leowilkin** [comment]
Thank you!

**Pull request merged**